docs/ops.html

```diff
@@ -3750,9 +3752,8 @@ <h3>Limitations</h3>
 
   <h3>Preparing for migration</h3>
   <p>
-    Before beginning the migration, the Kafka brokers must be upgraded to software version 3.5.0 and have the
-    "inter.broker.protocol.version" configuration set to "3.5". See <a href="#upgrade_3_5_0">Upgrading to 3.5.0</a> for
-    upgrade instructions.
+    Before beginning the migration, the Kafka brokers must be upgraded to software version {{fullDotVersion}} and have the
+    "inter.broker.protocol.version" configuration set to "{{dotVersion}}".
```

Comment:
Do we no longer need/want to link to the upgrade section?

Reply:
The link of the upgrade section use an id in the major_minor_fix format, for example upgrade_3_6_0. We currently don't have a variable holding the version in this format. I preferred not introducing another variable to update just for this. I also discarded changing the id to avoid breaking existing links.